app/models/cases.py

```diff
@@ -26,6 +26,9 @@
 class BaseCase:
     case_type: CaseTypes = Field(index=True)
 
+    def retrieve(self, session: Session, instance_id: uuid.UUID) -> SQLModel | None:
```

Comment:
There is already a retrieve method in the base request class https://github.com/ministryofjustice/laa-civil-case-api/blob/main/app/models/base.py#L82